app/processing/invoice-line/get-invoice-line.js

```diff
@@ -18,8 +18,8 @@ const getInvoiceLine = async (fundingCode, paymentRequestId) => {
 
   const { reductions, annualReduction } = await getReductions(fundingCode, paymentRequestId)
   const annualValue = grossValueInvoiceLine.value
-  const quarterlyValue = annualValue > MIN_PAYMENT_VALUE ? Math.trunc(annualValue * QUARTER) : MIN_PAYMENT_VALUE
-  const quarterlyReduction = Math.trunc(annualReduction * QUARTER)
```

Comment:
Delaying rounding of these values until after totals have been calculated.

That way totals are more accurate as not based on rounded values.